webhooks.go

```diff
@@ -0,0 +1,207 @@
+package twilio
+
+import (
+	"encoding/json"
+	"time"
+)
+
+type InteractionData struct {
+	Body string `json:"body"`
+}
+
+// https://www.twilio.com/docs/proxy/api/proxy-webhooks
+
+// https://www.twilio.com/docs/proxy/api/proxy-webhooks#callbackurl
+// These webhooks are fired for each new interaction and are informational only.
+type ProxyCallbackWebhook struct {
+	OutboundResourceStatus string    `form:"outboundResourceStatus"`
```

Comment:
I'm also not sure how this `form:""` struct tag works. Generally we handle these in the API with a `url.Values`.